Sources/AppcuesKit/AppcuesKit.docc/UniversalLinking.md

```diff
@@ -35,3 +35,27 @@ func application(_ application: UIApplication,
     // TODO: Parse the `incomingURL` here and return true if the link has been handled, false otherwise.
 }
 ```
+
+## Implementing a Domain Allow List 
+
+You may define a custom key, `AppcuesUniversalLinkHostAllowList`, in your _Info.plist_ file with an array of (string) host values to control which URLs are passed to `UIApplicationDelegate.application(_:continue:restorationHandler:)`. A URL whose host matches any of the provided host values will be treated as a potential universal link.
+
+An empty array will prevent all links from be treated as universal links and is functionally the same as calling ``Appcues/Config/enableUniversalLinks(_:)`` with a `false` argument.
```

Comment:
nice, was just thinking that merging that config in some fashion in the plist also would be nice.

Reply:
In the capacitor plugin we're disabling it by default which wouldn't really work with the plist approach.

https://github.com/appcues/appcues-capacitor-plugin/blob/d011729c86b1de005b5418bd66d2cfe81aa6143e/ios/Plugin/AppcuesPlugin.swift#L43-L46